src/modules/release/utils.ts

```diff
@@ -68,7 +68,8 @@ const runCommand = (
 ) => {
   let output
   try {
-    output = execSync(cmd, {stdio: hideOutput ? 'pipe' : ['inherit', 'pipe']})
+    console.log(`running ${cmd}`)
```

Comment:
is this console.log left here intentionally?

Reply:
Already removed. Tks